.golangci.yml

```diff
@@ -51,4 +52,4 @@ issues:
   exclude-rules:
     - path: policy/grammar\.go # ignore structtag issues there
       linters:
-        - govet
\ No newline at end of file
+        - govet
```

Comment:
By the way, we can drop this exclusion rule. Code of the placement policy parser has been moved to SDK a while ago.

Reply:
Fixed.